tasks/haskell-dependencies.yml

```diff
@@ -1,9 +1,5 @@
 ---
 - name: Install common Haskell build dependencies
   apt:
-    name: "{{ item }}"
+    name: "[ libpcre3-dev libsqlite3-dev libpq-dev ]"
```

Comment:
@speedlight is there any way to introduce this change without breaking backward compatibility? According to Ansible documentation, there are some [Special Variables](https://docs.ansible.com/ansible/latest/reference_appendices/special_variables.html), one of them is called `ansible_version` maybe we could use that one to pass the list of packages to be installed as a single argument or as a list of items?

Reply:
@speedlight on the other hand, if Ansible 2.4 is too old, maybe we could drop the support, but if that is the case I think we will need to update the role version accordingly and update the meta information regarding Ansible list of supported versions.

Reply:
@speedlight I don't think you need the quotes here since it is a list:

```yaml
apt:
  name:
    - libpcre3-dev
    - libsqlite3-dev
    - ...
```

Also you could declare it as a variable, although that adds one level of indirection:

```yaml
# defaults/main.yaml
sb_debian_base_haskell_dependencies:
  - libpcre3-dev
  ...

# tasks/haskell-dependencies.yaml
apt:
  name: "{{ sb_debian_base_haskell_dependencies }}"
```

I'm more inclined to the second option since it allows consumers to override the default value and add or remove dependencies based on their needs.

Reply:
Good point @sestrella So I think is better to define this initial packages with the variable in the `defaults.yml` file.